README.rst

```diff
@@ -115,6 +115,61 @@ Windows
     .\<your-env>\Scripts\activate
     pip install google-cloud-storage
 
+
+Tracing With OpenTelemetry
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a PREVIEW FEATURE: Coverage and functionality are still in development and subject to change.
+
+This library uses `OpenTelemetry`_ to generate traces on calls to Google Cloud Storage.
```

Comment:
Let's try "This library can be configured to use..." to make sure it's clear that it's optional for people not familiar with Otel.

Reply:
Good idea, updated!